docs/search/functions.md

```diff
@@ -378,7 +378,7 @@ Split the **input** string with the help of the specified **analyzer** into an a
 It has a regular return value unlike all other Search C8QL functions and is thus not limited to `SEARCH` operations. It is independent of Views. A wrapping `ANALYZER()` call in a search expression does not affect the _analyzer_ argument nor allow you to omit it.
 
 - **input** (string): text to tokenize
-- **analyzer** (string): name of an [Analyzer](../analyzers/index.md).
+- **analyzer** (string): name of an [Analyzer](analyzers/index.md).
```

Comment:
Are we not removing index.md from links anymore?

Reply:
@diana-macrometa It seems to depend on where we're linking, in this case it only works with index.md in the link.

At some point I plan to update the style guide to reflect whatever is going on with links now, but this is low on the list of priorities at the moment.